guides/doc-Configuring_virt-who_VM_Subscriptions/master.adoc

```diff
@@ -0,0 +1,24 @@
+include::common/attributes.adoc[]
+include::common/header.adoc[]
+:context: virt-who-vm-subs
+:virt-who-vm-subs:
+:ProductName: {ProjectServer}
+
+= {ConfiguringVMSubscriptionsDocTitle}
+
+// Render only for relevant and finished contexts
+ifdef::HideDocumentOnStable[]
+include::common/modules/snip_guide-not-ready.adoc[]
+endif::[]
+ifndef::HideDocumentOnStable[]
```

Comment:
I'd suggest we remove this. IMHO we've reviewed this when merging, so it should be good.

However, the virt-who plugin depends on Katello so I think we should hide the guide on `foreman-deb` and `foreman-el`. Just like we hide the managing content guide:
https://github.com/theforeman/foreman-documentation/blob/17deb9e38789346ab72cb31f1ef8f935f3561aa2/guides/doc-Managing_Content/master.adoc#L8-L16

And then hide the actual includes:
https://github.com/theforeman/foreman-documentation/blob/17deb9e38789346ab72cb31f1ef8f935f3561aa2/guides/doc-Managing_Content/master.adoc#L22

Reply:
Can you check up update?